flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,265 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+from future.moves.urllib.parse import urlparse
+
+import logging
+from collections import MutableSet
+
+import requests
+from requests import RequestException
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+
+log = logging.getLogger('ombi_list')
+
+class OmbiBase(object):
+
+
+    @staticmethod
+    def generate_series_id(season, episode=False):
+        tempid = 'S' + str(season.get('seasonNumber')).zfill(2)
+        if episode:
+            tempid = tempid + 'E' + str(episode.get('episodeNumber')).zfill(2)
+
+        return tempid
+            
+
+    @staticmethod
+    def generate_title(config, item, season=False, episode=False):
+        temptitle = item.get('title')
+
+        if item.get('releaseDate') and not config.get('strip_year'):
+            temptitle = temptitle + ' (' + str(item.get('releaseDate')[0:4]) +')'
+            
+        if season or episode:
+            temptitle = temptitle + ' ' + OmbiBase.generate_series_id(season, episode)
+            if episode:
+                if episode.get('title') and config.get('include_episode_title'):
+                    temptitle = temptitle + ' ' + episode.get('title')
+            
+        return temptitle
+
+    @staticmethod
+    def construct_url(config):
+        parsedurl = urlparse(config.get('base_url'))
+        if config.get('type') in ['movies']:
+            log.debug('Received movie list request')
+            return '%s://%s:%s%s/api/v1/Request/movie?apikey=%s' % (parsedurl.scheme, parsedurl.netloc, config.get('port'), parsedurl.path, config.get('api_key'))
+        elif config.get('type') in ['shows', 'seasons', 'episodes']:
+            log.debug('Received TV list request')
+            return '%s://%s:%s%s/api/v1/Request/tv?apikey=%s' % (parsedurl.scheme, parsedurl.netloc, config.get('port'), parsedurl.path, config.get('api_key'))
+        else:
+            raise plugin.PluginError('Error: Unknown list type %s.' % (config.get('type')))
+
+    @staticmethod
+    def get_json(url):
+        try:
+            return requests.get(url).json()
+        except RequestException as e:
+            raise plugin.PluginError('Unable to connect to Ombi at %s. Error: %s' % (url, e))
+
+    @staticmethod
+    def generate_entry(config, parent_request, child_request=False, season=False, episode=False):
```

Comment:
The default value for non-booleans should be `None`.

Reply:
Thanks, was going to go back and look a the python docks, was thinking null initially.